src/NuGet.Clients/PackageManagement.VisualStudio/ProjectSystems/LegacyCSProjPackageReferenceProjectProvider.cs

```diff
@@ -19,6 +19,9 @@ public class LegacyCSProjPackageReferenceProjectProvider : IProjectSystemProvide
     {
         private readonly IProjectSystemCache _projectSystemCache;
 
+        [Import]
+        private Lazy<ProjectSystem.IProjectServiceAccessor> ProjectServiceAccessor { get; set; }
```

Comment:
Even though this is lazy, the type argument for this coming from the CPS definitions assembly means that when your class is instantiated the CLR will also load at least that one CPS assembly. 
If you've fixed the RPS regression, fine. But if you want to ensure CPS doesn't load unless this lazy is actually evaluated, you may want to type this as `Lazy<object>` and set the contract name in your `[Import]` attribute so that you still get the value you want. Then cast to the CPS interface in a method that only executes once you know you're in a CPS scenario.